docs/concepts/pooling.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Ondat Storage Pooling"
+linkTitle: "Ondat Storage Pooling"
+weight: 1
+---
+
+## Overview
+
+> 💡 This feature is available in release `v2.9.0` or greater.
+
+Storage Pooling enables greater control over the type of storage a workload will use, this can be used to improve performance for storage-intensive applications and to reduce cost for less resource intensive applications.
+
+### What is Ondat Storage Pooling?
+
+The Storage Pooling feature allows creation of "Storage Pools", a Storage Pool is simply a group of drives connected to a set of machines.
+
+Applications can then use this Storage Pool, meaning any data wrote by the application will only be written to the drives in the Storage Pool
+
+In short storage pooling allows applications to target which drives they want to write data to.
```

Comment:
I would suggest to move this sentence up, so it is behind "...a set of machines." 

This should help users to understand what it does